### ISSUE TITLE
String values with double quotes cause a non-JSON output

### ISSUE THREAD

**Issue by jonmcoe** [chenckang/react-json-pretty#10]
`<JSONPrettyView json={{"key": 'My "value" has double quotes in it'}} />`

results in

```
{
  "key": "My "value" has double quotes in it"
}
```

<img width="827" alt="screen shot 2018-07-04 at 8 33 22 am" src="https://user-images.githubusercontent.com/5412533/42277327-fdcfdc32-7f64-11e8-9727-4f825bc18c1e.png">

which is no longer valid JSON.

It seems to me that the replace logic here is perhaps too aggressive https://github.com/chenckang/react-json-pretty/blob/master/src/JSONPretty.js#L47

**Comment by jonmcoe** [comment]
the removal of the second `.replace` like this:

```
diff --git a/src/JSONPretty.js b/src/JSONPretty.js
index 5e264d1..b6eefc2 100644
--- a/src/JSONPretty.js
+++ b/src/JSONPretty.js
@@ -44,7 +44,7 @@ module.exports = createReactClass({
       return text;
     }
 
-    return text.replace(/&/g, '&amp;').replace(/\\"([^,])/g, '&quot;$1').replace(/</g, '&lt;').replace(/>/g, '&gt;').replace(regLine, this._replace);
+    return text.replace(/&/g, '&amp;').replace(/</g, '&lt;').replace(/>/g, '&gt;').replace(regLine, this._replace);
   },
   render: function render() {
```

will give the desired

```
{
  "key": "My \"value\" has double quotes in it"
}
```

but i don't know the motivations for its inclusion to begin with

**Comment by chenckang** [comment]
The motivation is to replacing """ to "&quote;". However, as you mentioned, the replacing result should be "\\&quote;" namely "\\"".

I will fix it recently in the next patch.

**Comment by chenckang** [comment]
The problem is fixed in v1.7.9. 

Another reason of the replacement is that it can simplify the theme processing, for the theme processing must handle many double quotes.

**Comment by jonmcoe** [comment]
looks good to me. thanks for addressing so quickly!
